frontend/src/api/workflow/Workflow.ts

```diff
@@ -45,3 +45,8 @@ export async function importWorkflowConfigApi(
   const response = await axios.post(`${BASE_URL}/workflow/import`, formData)
   return response.data
 }
+
+export async function importSampleDataApi(): Promise<boolean> {
+  const response = await axios.get(`${BASE_URL}/sample_data`)
```

Comment:
`workspace_id` should be included in the request url.

Reply:
in the endpoint, `workflow/` is missing, too.

In `studio/app/common/routers/workflow.py`,
```python
router = APIRouter(prefix="/workflow", tags=["workflow"])

@router.get(
    "/sample_data",
    dependencies=[Depends(is_workspace_available)],
)
```
- the router has `prefix`, so the endpoint would be `/workflow/sample_data`